portal/src/main/webapp/js/src/oncoprint/OncoprintUtils.js

```diff
@@ -1623,7 +1623,7 @@ window.OncoprintUtils = (function() {
             "AMPLIFIED": "AMP",
             "DELETED": "DEL",
             "GAINED": "GAIN",
-            "HOMODELETED": "HOMDEL"
```

Comment:
I wonder if this file is deprecated because there is a file of the same name in the frontend:

https://github.com/cBioPortal/cbioportal-frontend/blob/2675e99f1eafdeeb0366ff1d18161f427c0fa3af/src/shared/components/oncoprint/OncoprintUtils.ts